plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Dusty Mabe <dusty@dustymabe.com>
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2020, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: rpm_ostree_pkg
+short_description: Install / Uninstall overlay additional packages
+version_added: "2.10"
+description:
+    - This module can be used to install / uninstall overlay additional packages using rpm-ostree command.
+options:
+    name:
+      description:
+      - Name of overlay package to install / remove.
+      required: true
+      type: list
+      elements: str
+      aliases: [ pkg ]
+    state:
+      description:
+      - state of the overlay package.
+      - C(present) and C(installed) will simply ensure that a desired package is installed.
+      - C(absent) and C(removed) will remove the specified package.
+      choices: [ 'absent', 'installed', 'present', 'removed' ]
```

Comment:
```suggestion
      choices: [ 'absent', 'present' ]
```
If `absent` and `removed` do the same, imo better not to use extra aliases